app/Friend.php

```diff
@@ -5,53 +5,21 @@
 use Illuminate\Database\Eloquent\Model;
 use App\Events\FriendWasBlacklisted;
 
-class Friend extends Model
+class Friend extends PhoneNumber
```

Comment:
@mattstauffer So I started on the constant stuff and the status trait but realized the friend and phone number are 95% the same, are you cool with doing it this way?